### PR TITLE
Docs for Import cert for controller APIM 3.1.0

### PR DESCRIPTION
**Pull request by 1akshitha**
Added the documentation on the Importing SSL certificates in the controller.

Related git issue: wso2/product-apim-tooling#417

This is to cover the doc impact of wso2/product-apim-tooling#235

### CODE REVIEW

File: en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md [hunk 1]
```diff
@@ -732,4 +732,20 @@ Run the following CTL command to change the default location of the export direc
...
+certificate of WSO2 API Manager is a self signed certificate and in production environments it is advised to use a
+different certificate than default.   
+
+If the certificate is the default WSO2 certificate or a CA signed signed certificate of a CA trusted by the OS, these
+certificates will be imported by default to the controller. If the CA or the certificate is new, you can add the
+certificate to the ```/home/.wso2apictl/certs``` directory. The certificates added to this directory will be imported 
+whenever an action is performed with the controller. Any DER or PEM encoded certificate can be used with controller.
+
+!!! Info
+    If you are using windows, CA certs will not be imported default and has to be added to the ```/home/.wso2apictl/certs```
```
Comment (wasuradananjith) [inline en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md:749]: Shall we add "by" in front of the word "default"?

File: en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md [hunk 2]
```diff
@@ -732,4 +732,20 @@ Run the following CTL command to change the default location of the export direc
         - Required :   
             `--export-directory`: Path to directory where APIs should be saved.   
             Default : `/home/.wso2apictl/exported`
+            
+            
+## Import SSL Certificate for Secure HTTP Communication with API Manager
+
+Different environments of API Manager can have different SSL certificates for secure HTTP communications. The default
+certificate of WSO2 API Manager is a self signed certificate and in production environments it is advised to use a
```
Comment (Chamindu36) [inline en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md:740]: "certificate of WSO2 API Manager is a self signed certificate and in production environments it is advised to use" shall we change this as follows
"certificate of WSO2 API Manager is a **self-signed** certificate and in production **environments, it is** advised to use "
```suggestion
certificate of WSO2 API Manager is a self-signed certificate and in production environments, it is advised to use a
```

File: en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md [hunk 3]
```diff
@@ -732,4 +732,20 @@ Run the following CTL command to change the default location of the export direc
...
+different certificate than default.
```
Comment (Chamindu36) [inline en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md:741]:
```suggestion
different certificate than the default.   
```

File: en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md [hunk 4]
```diff
@@ -732,4 +732,20 @@ Run the following CTL command to change the default location of the export direc
...
             Default : `/home/.wso2apictl/exported`
+            
+            
+## Import SSL Certificate for Secure HTTP Communication with API Manager
+
+Different environments of API Manager can have different SSL certificates for secure HTTP communications. The default
+certificate of WSO2 API Manager is a self signed certificate and in production environments it is advised to use a
+different certificate than default.   
+
+If the certificate is the default WSO2 certificate or a CA signed certificate of a CA trusted by the OS, these
```
Comment (Chamindu36) [inline en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md:743]:
```suggestion
If the certificate is the default WSO2 certificate or a CA-signed certificate of a CA trusted by the OS, these
```

File: en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md [hunk 5]
```diff
@@ -732,4 +732,20 @@ Run the following CTL command to change the default location of the export direc
...
+certificates will be imported by default to the controller. If the CA or the certificate is new, you can add the
+certificate to the ```/home/.wso2apictl/certs``` directory. The certificates added to this directory will be imported 
+whenever an action is performed with the controller. Any DER or PEM encoded certificate can be used with controller.
```
Comment (Chamindu36) [inline en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md:746]:
```suggestion
whenever an action is performed with the controller. Any DER or PEM encoded certificate can be used with the controller.
```

File: en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md [hunk 6]
```diff
@@ -732,4 +732,20 @@ Run the following CTL command to change the default location of the export direc
...
             Default : `/home/.wso2apictl/exported`
+            
+            
+## Import SSL Certificate for Secure HTTP Communication with API Manager
+
+Different environments of API Manager can have different SSL certificates for secure HTTP communications. The default
+certificate of WSO2 API Manager is a self signed certificate and in production environments it is advised to use a
+different certificate than default.   
+
+If the certificate is the default WSO2 certificate or a CA signed signed certificate of a CA trusted by the OS, these
```
[the remark this answers is not in the document]
Comment (1akshitha) [reply en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md]: Fixed

File: en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md (hunk 1 above)
Comment (1akshitha) [reply en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md]: Fixed

File: en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md (hunk 2 above)
Comment (1akshitha) [reply en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md]: Fixed

File: en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md (hunk 3 above)
Comment (1akshitha) [reply en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md]: Fixed

File: en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md (hunk 5 above)
Comment (1akshitha) [reply en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md]: Fixed

File: en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md (hunk 4 above)
Comment (1akshitha) [reply en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md]: Fixed

**Pull request merged**